modules/home-manager/gtk.nix

```diff
@@ -48,31 +45,68 @@ in
       default = [ "normal" ];
       description = "Catppuccin tweaks for gtk";
     };
-    gnomeShellTheme = mkEnableOption "Catppuccin gtk theme for GNOME Shell";
 
-    cursor = ctp.mkCatppuccinOpt "gtk cursors" // {
-      accent = ctp.mkBasicOpt "accent" cursorAccentType "gtk cursors";
-    };
+    gnomeShellTheme = mkEnableOption "Catppuccin gtk theme for GNOME Shell";
 
     icon = ctp.mkCatppuccinOpt "gtk modified Papirus icon theme" // {
       accent = ctp.mkAccentOpt "gtk modified Papirus icon theme";
     };
   };
 
-  config = lib.mkIf enable {
-    warnings = [
+  imports = [
+    (mkRemovedOptionModule
```

Comment:
my original thought was that i wanted to not limit the interface of the new option to this one as to avoid breaking changes after 1.0.0...but after thinking about it more, i really doubt that'll happen since these will probably stay for a good while 😆